roles-in-tech/machine-learning.md

```diff
@@ -10,13 +10,28 @@
 ### Prerequisites
 
 - The main prerequisite for machine learning is data analysis.
+  - Data analysis is a process of inspecting, cleansing, transforming, and modeling data with the goal of discovering useful information, informing conclusions, and supporting decision-making. Data analysis has multiple facets and approaches, encompassing diverse techniques under a variety of names, and is used in different business, science, and social science domains. In today's business world, data analysis plays a role in making decisions more scientific and helping businesses operate more effectively.
 
 ### Motivation
 
 - Machine learning(ML) is an application of artificial intelligence that provides systems the ability to automatically learn and improve from experience without being explicitly programmed.
 - Machine learning is given so much importance because it helps in predicting behavior and recognizing patterns that humans with their limited capacity cannot predict. It is important because every field wants automatic, adaptive, optimize and want to take the decision itself without any human interface.
 
-**Which companies use Machine Learning?** Everyone.  Some companies are [Pinterest](https://www.pinterest.com/), [Twitter](http://www.twitter.com/) and [Google Gmail](https://www.gmail.com).
+**Which companies use Machine Learning?** Everyone.  
+ Some companies are [Pinterest](https://www.pinterest.com/), [Twitter](http://www.twitter.com/) and [Google Gmail](https://www.gmail.com).
+ #### 1.Google
+  - Google is regarded by experts to be the most advanced company in the field of AI, machine learning and deep learning.
+The main reason for this is probably the amount of money the company has spent acquiring startups – Google has spent more than any other, according to CB Insights.
+Whether it’s something it acquired or developed in-house, Google last week launched an AI chatbot which answers your messages for you – like a more sophisticated auto-response email – in a range of contexts, including Skype, Slack and Twitter direct messages.
```

Comment:
`last week` is not good to use as many will read that after many days so they will think last week too at that moment of time.